### ISSUE TITLE
Error: TypeError: o[key].replaceAll is not a function durining report generation

### ISSUE THREAD

**Issue by hateshapepm** [pwndoc/pwndoc#220]
Awesome tool! I just updated everything and attempting to generate a report I get the following now:
![image](https://user-images.githubusercontent.com/73666452/144323832-e332070d-4ac9-46e9-8876-bb1aef2b6f49.png)

Used to have no issues. Thanks in advance.

**Comment by yeln4ts** [comment]
As it would be hard for me to reproduce could you maybe try a couple things ?
First you can try in the `report-generator.js` file at line 472 to change `replaceAll` by `replace` 
If it does not work, try to add `console.log(o[key])` just the line before and check the logs

Don't forget to rebuild the docker container

**Comment by lukewegryn** [comment]
Hey @yeln4ts! I am having the same issue, so I followed your instructions, and here is the output. If I just change to `replace` as you instructed, I get this error:

```
RangeError: Maximum call stack size exceeded
    at replaceSubTemplating (/app/src/lib/report-generator.js:466:30)
    at /app/src/lib/report-generator.js:470:22
    at Array.forEach (<anonymous>)
    at replaceSubTemplating (/app/src/lib/report-generator.js:468:35)
    at /app/src/lib/report-generator.js:470:22
    at Array.forEach (<anonymous>)
    at replaceSubTemplating (/app/src/lib/report-generator.js:468:35)
    at /app/src/lib/report-generator.js:470:22
    at Array.forEach (<anonymous>)
    at replaceSubTemplating (/app/src/lib/report-generator.js:468:35)
    at /app/src/lib/report-generator.js:470:22
    at Array.forEach (<anonymous>)
    at replaceSubTemplating (/app/src/lib/report-generator.js:468:35)
    at /app/src/lib/report-generator.js:470:22
    at Array.forEach (<anonymous>)
    at replaceSubTemplating (/app/src/lib/report-generator.js:468:35)
```

If I add in the login statement `console.log(o[key])` before the line with `replace`, I get

```
Company
Penetration Test
undefined
2021-12-17
2021-12-08
2021-12-17
company.com
Company
undefined
undefined
TypeError: o[key].replace is not a function
    at /app/src/lib/report-generator.js:473:30
    at Array.forEach (<anonymous>)
    at replaceSubTemplating (/app/src/lib/report-generator.js:468:35)
    at prepAuditData (/app/src/lib/report-generator.js:426:14)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Object.generateDoc (/app/src/lib/report-generator.js:17:24)
    at async /app/src/routes/audit.js:387:29
```

Any fix or work around would be greatly appreciated!

**Comment by yeln4ts** [comment]
Can you try that : https://github.com/pwndoc/pwndoc/issues/223#issuecomment-985650108 ?

**Comment by lukewegryn** [comment]
Thanks! Modifying the line you specified in https://github.com/pwndoc/pwndoc/issues/223#issuecomment-985650108 fixed it. Report export is working now.

**Comment by kosztyua** [comment]
I can also confirm by modifying both replaceAll to replace and #223 solved this issue.

**Comment by JulianRunnels** [comment]
I had the same issue. The fix above + #223 fixed the issue for me.
